ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb

```diff
@@ -8,18 +8,9 @@
     "\n",
```

Comment:
<p>I suggest adding more content here so this notebook can stand alone (not be reliant on the blog post), or adding a link to the blog post if you can get it ahead of time. </p><p> </p><p>If you do the former, you can copy and paste some of the introductory content from the blog post. Suggest covering:</p><ul><li>What metrics are used in the notebook</li><li>How you can create a labeling job using just those frame in which anomalies were detected to audit and correct those labels.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>